tools/cldr-apps/js/src/esm/cldrCoverageReset.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * cldrCoverageReset: encapsulate Survey Tool code for auto-resetting coverage
+ */
+import * as cldrCoverage from "../esm/cldrCoverage.js";
+import * as cldrGui from "../esm/cldrGui.js";
+import * as cldrLoad from "../esm/cldrLoad.js";
+import * as cldrMenu from "../esm/cldrMenu.js";
+import * as cldrText from "../esm/cldrText.js";
+
+import { notification } from "ant-design-vue";
+
+const TEST_DANGER = false; // false for production, true only when testing
```

Comment:
```suggestion
const TEST_DANGER = false && cldrStatus.getIsUnofficial(); // false for production, true only when testing
```

then even if it was set to `true` it would be false for production. 

https://unicode-org.atlassian.net/browse/CLDR-15239 is a better solution for such flags.